frontends/ol-components/src/components/RadioChoiceField/RadioChoiceField.tsx

```diff
@@ -44,8 +55,8 @@ const RadioChoiceField: React.FC<RadioChoiceFieldProps> = ({
   const labelId = useId()
   return (
     <FormControl className={className}>
-      <FormLabel id={labelId}>{label}</FormLabel>
-      <RadioGroup
+      <Label id={labelId}>{label}</Label>
+      <RadioGroupStyled
```

Comment:
Two notes on styling 

**First**
<img width="1036" alt="Screenshot 2024-07-16 at 12 06 41 PM" src="https://github.com/user-attachments/assets/36c5b960-18db-4500-aeab-8988e4a39a3e">

- The color here is coming from[ `theme.palette.primary.main`](https://github.com/mitodl/mit-open/blob/77752408e7b7d45f2712e7ca63ff771f74bcbe0a/frontends/ol-components/src/components/ThemeProvider/ThemeProvider.tsx#L47-L52)
    - We could main palette color to red. I briefly changed it to magenta (`rgba(255,0,255)`) just to make the effects more visible. I believe the only things using this color right now are radio buttons and Mui loading spinners.
    - Our color usage doesn't really align well with MUI, though. (Example: MUI uses `primary.main` for the color of buttons, radio groups, etc. But our buttons and our radio groups are different colors. *We don't use MUI's button component*.)
- Could also add  something like 
    ```
      ".MuiRadio-root.Mui-checked svg": {
        color: theme.custom.colors.red,
      },
    ```

**Second**
<img width="960" alt="Screenshot 2024-07-16 at 12 07 55 PM" src="https://github.com/user-attachments/assets/208651ad-bc09-4a97-83b5-e72ac40dab21">

The color here is coming from use of mui's FormLabel inside mui's FormControl. I don't think we want this styling. Or, if we do, we should have it everywhere (out of the scope of this PR).

*Aside: I used to not really understand the point of FormControl. Now I see what it does better.*

**Suggestion:** Remove FormControl and/or FormLabel (replace with div and/or label).

Reply:
I was able to tweak the `Checkbox` styles to match the design pretty closely, but the `RadioChoiceField` control just uses the MUI `Radio` control, which we haven't customized as of yet. We do have a `Radio` component in `ol-components`, but I believe it (as well as `Checkbox`) are old. If you replace the MUI `Radio` with our version in `RadioChoiceField`, it doesn't look right at all. The label should definitely be staying black though...